### PR TITLE
Template changes 202403.2.0

### PR DESCRIPTION
**Pull request by nnaik-conga**
End to end testing has been done.

### CODE REVIEW

File: src/app/auth/auth-config.module.ts [hunk 1]
```diff
@@ -12,14 +13,18 @@ export const httpLoaderFactory = () => {
...
     map((authOptions: AuthOptions) => {
+      let redirectUri = window.location.origin + window.location.pathname;
+      if(redirectUri.endsWith('/')) {
+        redirectUri = redirectUri.slice(0, -1);
+      }
       return {
         authority: authOptions.authEndpoint,
-        redirectUrl: window.location.origin + window.location.pathname,
-        postLogoutRedirectUri: window.location.origin + window.location.pathname,
+        redirectUrl: 'https://localhost:3000',
```
Comment (vbhan-conga) [inline src/app/auth/auth-config.module.ts:22]: We shouldn't hard code localhost urls, please take care of this.

**Review by dboppana-conga — APPROVED** [review]

**Comment by vbhan-conga** [comment]
@nnaik-conga please resolve merge conflicts.

File: src/app/auth/auth-config.module.ts [hunk 2]
```diff
@@ -12,14 +13,18 @@ export const httpLoaderFactory = () => {
 
   const authOptions$: Observable<OpenIdConfiguration> = config$.pipe(
     map((authOptions: AuthOptions) => {
+      let redirectUri = window.location.origin + window.location.pathname;
```
Comment (vbhan-conga) [inline src/app/auth/auth-config.module.ts:16]: @nnaik-conga This change is not required, please keep it consistent with ecommerce tempalte.

Comment (nnaik-conga) [reply src/app/auth/auth-config.module.ts:16]: This change was part of release-202403.2.0's the auth-config file. That's why we can see this change.

**Review by vbhan-conga — APPROVED** [review]

**Pull request merged**